src/app/(views)/github/t/@pullButtons/pulls/page.tsx

```diff
@@ -48,6 +93,25 @@ export default function Page() {
             handleSetIsOpenDialog(true)
           }}
         />
+
+        <Dropdown
+          type="radio"
+          data={{
+            groups: [{ label: '', items: values }],
+            value: '',
+            onValueChange: handleValueChange
+          }}
+        >
+          <Button
+            variant="ghost"
+            size="icon"
+            icon={
+              <Tooltip tooltip={'NOT DECIDED'}>
+                <SettingsIcon className="w-full h-full" />
+              </Tooltip>
+            }
+          />
```

Comment:
What is this button's purpose?

Reply:
Hmm? What do you mean? That just makes the SettingIcon work. I was wondering,,, Am I wrong?